content/guides/template-gds/_index.md

```diff
@@ -0,0 +1,227 @@
+---
+title: 'Create GDS from Tinytapeout Git Templates'
+linkTitle: 'Template to GDS'
+description: 'preparing your design for submission using the TinyTapeout GitHub template'
+weight: 60
+---
+
+
+This guide will walk you through the process of preparing your design for submission using the TinyTapeout GitHub template. We’ll cover the necessary steps to convert your design into the ASIC files required for chip manufacturing and also view your files with the 3d viewer.
+If you don't already have a GitHub account, please create one before proceeding.
+
+## Pre-requisites
+
+**Check Wokwi Project**: The digital design that you want to tapeout can be created following this [video tutorial](https://tinytapeout.com/digital_design/wokwi/). Note that your wokwi project should comply with the [Tiny Tapeout Wokwi template](https://wokwi.com/projects/354858054593504257).
```

Comment:
wokwi -> Wokwi

Please be consistent with the naming. You also have a WOKWI further down

Reply:
Use a numbered list and get rid of the ** bold formatting